streams/writable-streams/aborting.js

```diff
@@ -442,10 +439,10 @@ promise_test(() => {
     writer.close();
     const abortPromise = writer.abort();
     return flushAsyncEvents().then(() => {
-      assert_false(abortCalled, 'underlying abort should not be called while close is in-flight');
+      assert_array_equals(ws.events, ['close'], 'abort should not be called while close is pending');
       resolveClose();
       return abortPromise.then(() => {
-        assert_false(abortCalled, 'underlying abort should not be called after close completes');
+        assert_array_equals(ws.events, ['close'], 'abort should not be called while close is pending');
```

Comment:
close is no longer pending here. How about just removing "while close is pending" from the description?